test/libgit2.jl

```diff
@@ -915,4 +915,75 @@ mktempdir() do dir
         end
     end
     =#
+
+    @testset "Hostname verification" begin
+        openssl_command = ""
+        if is_linux()
+            try
+                # OpenSSL needs to be on the path
+                openssl_command = strip(readstring(`which openssl`))
```

Comment:
why not something trivial like `openssl version` instead of relying on `which` ?